lib/ansible/modules/storage/purestorage/purefa_pgsnap.py

```diff
@@ -31,13 +32,18 @@
   state:
     description:
     - Define whether the protection group snapshot should exist or not.
```

Comment:
`Define whether the protection group snapshot should exist or not. I(copy) added in Ansible 2.7`
Also does `copy` need explaining at all, what is it copying from?

If you do `state: present` to create a protection group snapshot then do `state: copy` can you change it? (Not sure how how this would be used)

Reply:
Added some commentary